spring-cloud-dataflow-server-core/pom.xml

```diff
@@ -170,6 +170,11 @@
 			<groupId>org.flywaydb</groupId>
 			<artifactId>flyway-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>io.projectreactor</groupId>
+			<artifactId>reactor-core</artifactId>
+			<version>3.3.0.RELEASE</version>
```

Comment:
We definitely don't want to be explicit about versioning as it should come from the underlying dependency management. If there is any specific reason, could you elaborate why to do that and if so, we can move the version to a variable for better management.